azure-iot-device/azure/iot/device/iothub/pipeline/edge_pipeline.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class EdgePipeline(object):
```

Comment:
>EdgePipeline [](start = 6, length = 12)

I still think this name is confusing, but I'm not sure what's better.  

EdgeHTTPPipeline?  (nah, don't need HTTP, but there must be _some_ way to communicate why this pipeline is different.
EdgeSpecificFeaturePipeline?
EdgeMethodInvokePipeline?

Luckily this is internal, so we have a little more freedom to rename in the future.
FWIW, the methods in this pipeline are handled by the EdgeHub module, so there is some symmetry if we call this EdgeHubPipeline to mirror IoTHubPipeline
Or, if we're being pedantic, maybe this is EdgeHub pipeline because it's only useful with EdgeHub, but the other one is HubPipeline since it can be used with EdgeHub or IoTHub?